File: cloud-arm/src/main/java/com/sequenceiq/cloudbreak/cloud/arm/context/NetworkInterfaceCheckerContext.java
package com.sequenceiq.cloudbreak.cloud.arm.context;

import com.sequenceiq.cloudbreak.cloud.arm.view.ArmCredentialView;

public class NetworkInterfaceCheckerContext extends GroupNameContext {

    private String networkName;

    public NetworkInterfaceCheckerContext(ArmCredentialView armCredentialView, String groupName, String networkName) {
        super(armCredentialView, groupName);
        this.networkName = networkName;
    }

    public String getNetworkName() {
        return networkName;
    }
}

File: cloud-arm/src/main/java/com/sequenceiq/cloudbreak/cloud/arm/context/NetworkInterfaceCheckerContext.kt
package com.sequenceiq.cloudbreak.cloud.arm.context

import com.sequenceiq.cloudbreak.cloud.arm.view.ArmCredentialView

class NetworkInterfaceCheckerContext(armCredentialView: ArmCredentialView, val groupName: String, val networkName: String) : ArmStatusCheckerContext(armCredentialView)
